lib/matplotlib/tests/test_path.py

```diff
@@ -77,9 +78,33 @@ def test_exact_extents(path, extents):
     # the way out to the control points.
     # Note that counterintuitively, path.get_extents() returns a Bbox, so we
     # have to get that Bbox's `.extents`.
+    path, extents = precomputed_curve.path, precomputed_curve.extents
     assert np.all(path.get_extents().extents == extents)
 
 
+@pytest.mark.parametrize('precomputed_curve', _test_curves)
+def test_signed_area(precomputed_curve):
+    path, area = precomputed_curve.path, precomputed_curve.area
+    assert np.isclose(path.signed_area(), area)
+    # now flip direction, sign of *signed_area* should flip
+    rverts = path.vertices[:0:-1]
+    rverts = np.append(rverts, np.atleast_2d(path.vertices[0]), axis=0)
```

Comment:
Is this not?
```suggestion
    rverts = path.vertices[::-1]
```

Reply:
yup, this leftover (and typo'd) code deals with reversing paths that have a CLOSEPOLY. 

should read 

```
rverts = np.append(rverts, np.atleast_2d(path.vertices[0]), axis=0)
```

since our tests don't need it, I removed this and will implement a "correct" `Path.reversed()` in the future if needed.